dev-libs/libpurple/libpurple-2.13.0.recipe

```diff
@@ -0,0 +1,159 @@
+	devel:libgnt$secondaryArchSuffix = $libgntVersionCompat
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}_devel
+	libpurple$secondaryArchSuffix == $portVersion base
+	"
+PROVIDES_finch="
+	libpurple${secondaryArchSuffix}_finch = $portVersion
+	cmd:finch$secondaryArchSuffix
+	"
+REQUIRES_finch="
+	haiku${secondaryArchSuffix}
+	libpurple$secondaryArchSuffix == $portVersion base
+	lib:libffi$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgmodule_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libgthread_2.0$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libidn$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libpython2.7$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgobject_2.0$secondaryArchSuffix
+	devel:libgmodule_2.0$secondaryArchSuffix
+	devel:libgthread_2.0$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libidn$secondaryArchSuffix
+	devel:libnss3$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libnspr4$secondaryArchSuffix
+	devel:libfreebl3$secondaryArchSuffix
+	devel:libnss3$secondaryArchSuffix
+	devel:libnssckbi$secondaryArchSuffix
+	devel:libnssdbm3$secondaryArchSuffix
+	devel:libnssutil3$secondaryArchSuffix
+	devel:libsmime3$secondaryArchSuffix
+	devel:libsoftokn3$secondaryArchSuffix
+	devel:libssl3$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:gettext
+	cmd:intltool_update
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python2
+	cmd:sed
+	"
+
+PATCH()
+{
+	sed -i 's/resolv/network/g' configure.ac
+}
+
+defineDebugInfoPackage libpurple$secondaryArchSuffix \
+	"$libDir"/libpurple.so.$libpurpleVersion \
+	"$libDir"/libgnt.so.$libgntVersion
+
+BUILD()
+{
+	autoreconf -fi
+	CFLAGS="-D__SVR4 -D__sun -D_XPG5 -D_BSD_SOURCE" \
+	LDFLAGS="-lbsd" \
+	runConfigure ./configure \
+		--disable-gtkui --disable-screensaver \
+		--disable-meanwhile --disable-dbus --disable-gstreamer --disable-vv \
+		--disable-avahi --disable-gnutls --disable-tcl -enable-textui \
+		--with-nspr-includes=$(finddir B_SYSTEM_HEADERS_DIRECTORY)/${secondaryArchSubDir} \
+		--with-nspr-libs=$(finddir B_SYSTEM_LIB_DIRECTORY)/${secondaryArchSubDir} \
+		--with-nss-includes=$(finddir B_SYSTEM_HEADERS_DIRECTORY)/${secondaryArchSubDir}/nss \
+		--with-nss-libs=$(finddir B_SYSTEM_LIB_DIRECTORY)/${secondaryArchSubDir} \
+		--with-ncurses-headers=$(finddir B_SYSTEM_HEADERS_DIRECTORY)/${secondaryArchSubDir} \
+		--disable-dependency-tracking
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	rm -rf $prefix/share $libDir/lib*.la
+
+	fixPkgconfig
+
+	packageEntries finch $binDir
```

Comment:
```diff
-	packageEntries finch $binDir
+	packageEntries finch \
+		$binDir \
+		$manDir
```
(This will move `/system/documentation/man/man1/finch.1` to the libpurple_finch subpackage.)